translate/src/modules/translationform/components/TranslationForm.tsx

```diff
@@ -1,27 +1,31 @@
 import { Localized } from '@fluent/react';
-import React, { useCallback, useContext, useLayoutEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 
-import { EditorData, useEditorValue } from '~/context/Editor';
+import { EditFieldHandle, EditorData } from '~/context/Editor';
 import { EntityView } from '~/context/EntityView';
 import { Locale } from '~/context/Locale';
 import { usePluralExamples } from '~/hooks/usePluralExamples';
 import { searchBoxHasFocus } from '~/modules/search/components/SearchBox';
 import { CLDR_PLURALS } from '~/utils/constants';
 
 import { EditAccesskey } from './EditAccesskey';
-import { EditField, EditFieldProps } from './EditField';
+import { EditField } from './EditField';
 import './TranslationForm.css';
 
 const Label = ({
   getExample,
-  htmlFor,
   labels,
 }: {
   getExample: (label: string) => number | undefined;
-  htmlFor: string;
   labels: Array<{ label: string; plural: boolean }>;
 }) => (
-  <label htmlFor={htmlFor}>
+  <label>
```

Comment:
Sadly, HTML `<label> and `contentEditable` doesn't really play well together, so clicking on the label will no longer focus the appropriate field.

Reply:
That's a nasty UX issue. Why do we need `contenteditable`?

Reply:
Because it's used by CodeMirror, and effectively the basis on which any solution other than `<textarea>` is built.

Reply:
Oh, ok - I thought we were also using `contenteditable` in the label.